segmentstore/server/src/main/java/io/pravega/segmentstore/server/containers/StreamSegmentContainer.java

```diff
@@ -177,6 +180,29 @@ private MetadataStore createMetadataStore() {
         return builder.build();
     }
 
+    /**
+     * Initializes storage.
+     *
+     * @throws Exception
+     */
+    private void initializeStorage() throws Exception {
+        this.storage.initialize(this.metadata.getContainerEpoch());
+
+        if (this.storage instanceof ChunkedSegmentStorage) {
+            ChunkedSegmentStorage storageManager = (ChunkedSegmentStorage) this.storage;
+
+            // Initialize storage metadata table segment
+            ContainerTableExtension tableExtension = getExtension(ContainerTableExtension.class);
+            Preconditions.checkNotNull(tableExtension);
```

Comment:
And what it if it is null? Won't this be checked in the constructor of TableBasedMetadataStore below?

Reply:
removed.